apps/src/music/blockly/FieldSounds.js

```diff
@@ -69,6 +70,10 @@ class FieldSounds extends GoogleBlockly.Field {
   }
 
   showEditor_() {
+    if (this.showingEditor) {
```

Comment:
A bit surprising that Blockly doesn't handle this automatically - I would think it would call the `dropdownDispose` handler whenever the field is reopened?

Reply:
the show editor logic is pretty [bare bones](https://github.com/google/blockly/blob/81e2203f7fd4f61d015bd6e82114c69da9656adf/core/field.ts#L778-L791), they expect the subclass to define `showEditor_`